README.md

```diff
@@ -1,75 +1,168 @@
+
+Once you have your API key:
+
+```sh
+export FOSSA_API_KEY=XXXX # Use your API key here.
 ```
 
-## Contributing
+#### Running an analysis
+
+Now we can run an analysis. To run an analysis, all you need to do is navigate to your project's directory and run `fossa analyze`.
 
-For development documentation (still WIP, but not empty), see our [Development
-Docs Homepage](devdocs/index.md).
+**NOTE:** While `fossa` will try its best to report available results for any kind of project, you'll get the best results by running in a directory with a working project build. A working build lets us integrate directly with your build tool to identify dependencies, instead of trying to infer dependencies from your source code.
+
+```sh
+$ cd $MY_PROJECT_DIR # Use your actual project location here.
+
+$ fossa analyze
+[ INFO] Using project name: `https://github.com/fossas/spectrometer`
+[ INFO] Using revision: `09ca72e398bb32747b27c0f43731678fa42c3c26`
+[ INFO] Using branch: `No branch (detached HEAD)`
+[ INFO] ============================================================
+
+      View FOSSA Report:
+      https://app.fossa.com/projects/custom+1%2fgithub.com%2ffossas%2fspectrometer/refs/branch/master/09ca72e398bb32747b27c0f43731678fa42c3c26
+
+  ============================================================
+```
+
+#### Viewing your results
+
+Once an analysis has been uploaded, you can view your results in the FOSSA web application. You can see your analysis by using the link provided as output by `fossa analyze`, or by navigating to your project and revision in the FOSSA web application.
+
+#### What next?
+
+Now that your analysis is complete, there are a couple things you might want to do after an initial integration:
+
+- **Double-check your results.** Some analysis methods may produce partial or unexpected results depending on what information was available when you ran the analysis. If something seems wrong, [our debugging guide](./docs/walkthroughs/debugging-your-integration.md) can help you diagnose and debug your integration.
+
+- **Scan for issues and generate a compliance report.** Once your analysis is ready, we'll automatically queue an issue scan and report the results in the web application. If you want to try out different issue scanning configurations, you can [change your policy](TODO) or [run a manual issue scan](TODO). Once an issue scan is complete, you can also [generate a report](TODO) from the web application.
+
+- **Set up FOSSA in your CI.** You can also use your issue scan results as inputs to CI scripts. For GitHub repositories, you can use FOSSA's [native GitHub integration](TODO) to report a status check on your PRs. For other CI integrations, you can also [use `fossa test`](TODO) to get programmatic issue status in CI.
+
+### Using Spectrometer as a standalone tool
```

Comment:
I don't see a great reason to include this section. I'd prefer to omit this section rather than field any questions from customers who get the idea that they want to try this. I'm open to hearing why this is beneficial though.